datadog_checks_dev/datadog_checks/dev/tooling/commands/release.py

```diff
@@ -584,6 +584,17 @@ def make(ctx, check, version, initial_release, skip_sign, sign_only):
         if version:
             # sanity check on the version provided
             cur_version = get_version_string(check)
+
+            # Keep track of intermediate version bumps
+            prev_version = cur_version
+            for method in version.split(','):
+                # Apply any supported version bumping methods. Chaining is required for going
+                # from mainline releases to development releases since e.g. x.y.z > x.y.z-rc.A.
+                # So for an initial bug fix dev release you can do `fix,rc`.
+                if method in VERSION_BUMP:
+                    version = VERSION_BUMP[method](prev_version)
```

Comment:
Can you explain? I don't understand

Reply:
Well according to https://python-semver.readthedocs.io/en/stable/api.html#semver.bump_prerelease, you can specify a token for the kind of prerelease you do. The default is `rc`.
So with the current implementation, even if you do `dev release make vsphere fix,dev`, you will end up with the version going from e.g `3.4.0` to `3.4.1-rc.1` instead of `3.4.1-dev.1` as one could intuitively expect.